packages/fuels-abigen-macro/tests/harness.rs

```diff
@@ -1560,6 +1559,46 @@ async fn test_wallet_balance_api() {
     }
```

Comment:
please rework the comments l45-60 as they are outdated, especially with your new additions

Reply:
I believe the ''clean code" approach is to not have these kinds of comments in code at all as they are not maintainable.
This information should be provided by the book (maybe is already).

Reply:
Agreed!